src/core/MeshGenerator.ts

```diff
@@ -137,21 +132,22 @@ export class MeshGenerator {
       let attachmentColor: Color = null;
       const z = zSpacing * slotIndex;
       let numFloats = 0;
-      let vertexSize = _clipper.isClipping() ? 2 : MeshGenerator.VERTEX_SIZE;
+      const isClipping = _clipper.isClipping();
+      let vertexSize = isClipping ? 2 : MeshGenerator.VERTEX_SIZE;
       if (attachment instanceof RegionAttachment) {
         let regionAttachment = <RegionAttachment>attachment;
         attachmentColor = regionAttachment.color;
         vertices = this._vertices;
```

Comment:
need set every time ?

Reply:
Yes. Clipper.isClipping will return true when clipper.clipStart function called.
During the animation playback, you don't know when the clipping will occur or which slot will be clipped, so it needs to be checked in a loop.